go-controller/pkg/node/bridgeconfig/bridgeconfig_testutil.go

```diff
@@ -50,7 +50,7 @@ func CheckUDNSvcIsolationOVSFlows(flows []string, netConfig *BridgeUDNConfigurat
 		protoPrefix = "ip"
 	} else {
 		mgmtMasqIP = netConfig.V6MasqIPs.ManagementPort.IP.String()
-		protoPrefix = "ip6"
+		protoPrefix = "ipv6"
```

Comment:
Thinking how we could avoid this in the feature...
Can we define a constant to be used instead of a magic string that we will like get wrong again in the future?

Reply:
yeah good point. I will change this to a constant.